### PR TITLE
Refactor notify-issues

### PR DESCRIPTION
**Pull request by fepitre**
Refactor notify-issues and use `NotifyIssue` class directly by `github-action` instead of subprocess.

### CODE REVIEW

File: utils/notify_issues.py [hunk 1]
```diff
@@ -320,133 +412,191 @@ def notify_closed_issues(
...
+        add_labels, delete_labels = self.get_labels(
+            command=command,
+            repository_type=repository_type,
+            build_status=build_status,
+            dist_label=dist_label,
+            package_name=package_name,
+        )
+
+        if command == "upload":
+            build_status = "uploaded"
```
Comment (marmarek) [inline utils/notify_issues.py]: This is wrong, it will notify failed upload as successful one.

**Review by marmarek — CHANGES_REQUESTED** [review]
Te comment below is about the case we found broken recently. Maybe add also a test for failed upload (both component and template)?

File: utils/notify_issues.py (hunk 1 above)
Comment (fepitre) [reply utils/notify_issues.py]: Yes indeed I did not catched it in the refactor thank you. Let me fix that.

**Review by marmarek — APPROVED** [review]
Approved as of ca700c10b36ab2d6d8eeefbd9469651024d82324

**Pull request merged**